.travis.yml

```diff
@@ -1,5 +1,6 @@
 ---
-dist: bionic
```

Comment:
is there a reason for switching from ubuntu 18 to 16?

Reply:
clearly I'm not keeping up with the times :facepalm: